### ISSUE TITLE
Increase width of rules

### ISSUE THREAD

**Issue by Gitoffthelawn** [einaregilsson/Redirector#37]
When displaying the rules, the width is so narrow that long rules have to wrap onto multiple lines, making them hard to read.

Recommendation: Increase the width.

If you would like me to do it, just ask or assign it to me, or both!  :-)

**Comment by einaregilsson** [comment]
Not sure about that one. I generally don't like things being too wide, I think it makes things harder to read, not easier. Do you have many rules that are wider than the current width?

**Comment by Gitoffthelawn** [comment]
I generally don't like things being too wide either.  But for URLs and regex, it's much more readable when they are not wrapped across two lines.

With v3.0.4, about half my rules have a URL or regex that is force-wrapped onto two lines.

**Comment by einaregilsson** [comment]
Ok, I'll see what I can do. I'll do a cleanup release in a couple of weeks,
gather up all the small issues people find after upgrading and try to fix
all or most of them.

**Comment by Gitoffthelawn** [comment]
Sounds great.  BTW, drag and drop is intentionally gone and replaced with the arrows, correct?

**Comment by einaregilsson** [comment]
Correct

**Comment by Gitoffthelawn** [comment]
Thanks :-)

**Comment by Gitoffthelawn** [comment]
One thing that will really help readability is to not split URL's onto multiple lines, unless absolutely necessary.

For each rule's example, it is common for the "from" URL to be on one line, followed by the right-arrow, followed by the "to" URL half on the same line and half on the next line.  In such cases, I recommend placing the "from" URL on one line, immediately followed by the right-arrow.  On the line below it, show the "to" URL.  This way, the "from" URL and "to" URL will be horizontally aligned, making them much easier to compare.

**Comment by einaregilsson** [comment]
Could be hard to do that in CSS. Maybe I can put a span around the url with white-space:nowrap. Will look at it when I do the next version.

**Comment by Gitoffthelawn** [comment]
Yes, I played around with the CSS and it was hard to accomplish.  I think adding SPANs will do the job much more effectively.

**Comment by Golutazem** [comment]
For me and presumably some other users, it makes things easier to use and read if they are wide than if things get cropped or wrapped.

An option for us users to display using full available width would be helpful.
